src/_gettsim/transfers/rente.py

```diff
@@ -19,88 +19,224 @@ def sum_ges_rente_priv_rente_m(priv_rente_m: float, ges_rente_m: float) -> float
     return out
 
 
-@policy_info(params_key_for_rounding="ges_rente")
-def ges_rente_vor_grundr_m(
-    ges_rente_zugangsfaktor: float,
-    entgeltp_ost_update: float,
-    entgeltp_west_update: float,
+@policy_info(end_date="2020-12-31")
+def ges_rente_m(bruttorente_m: float) -> float:
+    return bruttorente_m
```

Comment:
Why do we nee the `is_rentner` check in the function directly below, but not here? Should be consistent.